Cesium.CodeGen/Contexts/IDeclarationScope.cs

```diff
@@ -26,4 +28,16 @@ internal interface IDeclarationScope
     /// </summary>
     /// <param name="identifier">Label to add to the current scope.</param>
     void AddLabel(string identifier);
+
+    /// <summary>
+    // Registers child declaration scope.
+    /// </summary>
+    /// <param name="childScope">Child scope to add to the current one.</param>
+    void RegisterChildScope(IDeclarationScope childScope);
```

Comment:
What's the use of this? It's never called.

Reply:
Ooh yeah. I should remove this. This part I’m not even sure how it would be used.